packages/engine/README.md

```diff
@@ -339,9 +339,9 @@ If you haven't created and exported a project from [hCore], it's also possible t
 
 At the end of each simulation run, various outputs appear within the `./<OUTPUT FOLDER>/<PROJECT NAME>/<EXPERIMENT NAME>/<EXPERIMENT ID>/<SIMULATION ID>` directories.
 
-Where `<EXPERIMENT ID>` and `<SIMULATION ID>` are unique identifiers created for each run of an experiment or simulation.
+Where `<PROJECT NAME>` is the name of the folder containing your experiments, and, `<EXPERIMENT ID>` and `<SIMULATION ID>` are unique identifiers created for each run of an experiment or simulation.
```

Comment:
```suggestion
Where `<PROJECT NAME>` is the name of the folder containing your experiments, and `<EXPERIMENT ID>` and `<SIMULATION ID>` are unique identifiers created for each run of an experiment or simulation.
```

Reply:
I started writing some nonsense about Oxford commas and breaking this expression into two clauses with the additional comma but have decided to just introduce bulletpoints 😂